sktime/annotation/base/_base.py

```diff
@@ -681,18 +681,18 @@ def change_points_to_segments(y_sparse, start=None, end=None):
         [5, 7)    3
         dtype: int64
         """
-        breaks = y_sparse.values
+        breaks = y_sparse.tolist()
```

Comment:
these changes seem to cause failures in non-suite tests - can you explain what this is doing?